pkg/storage/unified/sql/backend.go

```diff
@@ -6,15 +6,22 @@ import (
 	"errors"
 	"fmt"
 	"math"
+	"net/http"
 	"sync"
 	"time"
 
+	"cloud.google.com/go/spanner"
```

Comment:
We have a separate extension for spanner in grafana-enterprise, this class here won't be called for spanner

Reply:
On top of that, we should also not depend on spanner packages from OSS grafana as general rule.